tests/wpt/mozilla/tests/mozilla/referrer-policy/generic/subresource/mozresource.py

```diff
@@ -83,6 +84,9 @@ def respond(request,
     if access_control_allow_origin != None:
         response.writer.write_header("access-control-allow-origin",
                                      access_control_allow_origin)
+    if http_header_referrer_policy != None:
+        response.writer.write_header("Referrer-Policy",
```

Comment:
Let's make this `referrer-policy` to match the style of the other headers in this file.